app/views/editor/level/thangs/LevelThangEditView.coffee

```diff
@@ -46,7 +47,8 @@ module.exports = class LevelThangEditView extends CocoView
       supermodel: @supermodel
       level: @level
       world: @world
-      thangType: thangType
+
+    if @level.isType('hero', 'hero-ladder', 'hero-coop', 'course', 'course-ladder', 'game-dev', 'web-dev') or utils.isCodeCombat then options.thangType = thangType
```

Comment:
why do we need this flag? Seems both coco and ozar have code. Am i missing something?